.cirrus.yml

```diff
@@ -23,6 +23,16 @@ timeout_in: 60m
 task:
     name: "Validate"
     alias: "validate"
+    # Ref: https://cirrus-ci.org/guide/writing-tasks/#supported-functions
+    skip: &notdocs <-
+        !changesInclude(
+          'base_images/***',
+          'cache_images/***',
+          'ci/***',
+          'image_builder/***',
+          'podman/***',
+          '*.sh',
+          'Makefile')
```

Comment:
...and `.cirrus.yml`.

I really don't think this is a wise approach; I would try filing a Cirrus PR asking sweetly for `changesAreOnly(...)`

Reply:
ya, this would be a PITA to maintain, even if it worked.

Reply:
TBH, it's probably a decision best left up to humans.  I've never had good experiences with these kinds of execution-conditions.  There always seems to be some corner-case or another that breaks it.  Whereas just sticking `[CI:DOCS]` or similar in the title, both functions as directed and leaves a record of what happened (in the logs).